Makefile

```
@@ -0,0 +1,202 @@
+	@echo "✓ General setup complete!"
+	@echo ""
+	@echo "For Envoy gateway development, also run: make setup-envoy"
+
+# Envoy gateway development setup
+setup-envoy: submodule-init proto-compile
+	@echo ""
+	@echo "✓ Envoy gateway setup complete!"
+	@echo "You can now run: make envoy-stack"
+
+# Restore proto submodules to pinned commits from .gitmodules
+upgrade-protos:
```

checkmake (0.3.2)

[warning] Target body for "upgrade-protos" exceeds allowed length of 5 lines (16).

(maxbodylength)